src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/BuildUpToDateCheck.cs

```diff
@@ -93,7 +93,7 @@ private void Log(LogLevel level, string message, params object[] values)
             IProjectSystemOptions projectSystemOptions,
             ConfiguredProject configuredProject,
             Lazy<IFileTimestampCache> fileTimestampCache,
-            IProjectAsynchronousTasksService tasksService,
+            [Import(ExportContractNames.Scopes.ConfiguredProject)] IProjectAsynchronousTasksService tasksService,
```

Comment:
Well, it was working on my machine and testing before I checked in, but I haven't determined why. It definitely does not work now.